<?xml version="1.0"?>
<launch>

<node pkg="user_server" type="user_server_node" name="user_server_node" output="screen">
    <param name="file_location" value="$(find user_server)/launch/mockmap0.pcd" />
    <param name="tracker_timer_hz" value="5.0"/>
    <param name="cloud_timer_hz" value="12.0"/>
    <param name="single_or_multi" value="multi"/>
    <param name="formation" value="left-right-facing"/>
    <param name="agents" value="2"/>
    <rosparam param="waypoints">
        [0.0, -30.0, 2.0, 
        0.0, 20.0, 2.0,
        0.0, -30.0, 2.0,
        0.0, 20.0, 2.0]</rosparam>
</node>

<include file="$(find user_server)/launch/generated_left-right-facing_2.launch" />

<node name="rviz" pkg="rviz" type="rviz" args="-d $(find quad)/rviz/generated_display_2.rviz"/>

</launch>